funders/nsf-grants/geo-ose.md

```diff
@@ -1,6 +1,3 @@
----
```

Comment:
I didn't see a need for this or the other smaller grants to be in the TOC - am I missing something?

Reply:
Currently, grants are placed in the subnav under the NSF item (so you have to click on the NSF item to see them). The `orphan: true` doesn't remove it from the navigation or TOC. I'm not sure jupyter-book supports this but I can look into it.